src/pages/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,681 @@
+          <p class="banner">Astro has {{stars}} 🧑‍🚀</p>
+          <Footer />
+        </div>
+      </div>
+    </template>
+
+    <script>
+    import Vue from 'vue'
+    
+    export default Vue.extend({
+      name: 'IndexPage',
+      async asyncData() {
+        const res = await fetch('https://api.github.com/repos/withastro/astro')
+        const json = await res.json();
+        return {
+          message: json.message,
+          stars: json.stargazers_count || 0,
+        };
+      }
+    });
+    </script>
+    
+    <style scoped>
+    .banner {
+      background-color: #f4f4f4;
+      padding: 1em 1.5em;
+      text-align: center;
+      margin-bottom: 1em;
+    }
+    <style>
+    ```
+  </Fragment>
+  <Fragment slot="astro">
+    ```astro title="Page.astro"
+    ---
+    import Header from "./header";
+    import Footer from './footer';
+    import "./layout.css";
+
+    const res = await fetch('https://api.github.com/repos/withastro/astro')
+    const json = await res.json()
+    const message = json.message;
+    const stars = json.stargazers_count || 0;
+    ---
+    
+    {message === "Not Found" ? 
+          <p>The repository you're looking up doesn't exist</p> :
+          <>
+                <Header />
+                <p class="banner">Astro has {stars} 🧑‍🚀</p>
+                <Footer />
+            </> 
+    }
+    
+    <style>
+      .banner {
+        background-color: #f4f4f4; 
+        padding: 1em 1.5em;
+        text-align: center;
+        margin-bottom: 1em;
+      }
+    <style>
+    ```
+  </Fragment>
+</AstroVueTabs>
+
+### Migrating Layout Files
+
+You may find it helpful to start by converting your Nuxt layouts and templates into [Astro layout components](/en/core-concepts/layouts/).
+
+Each Astro page explicitly requires `<html>`, `<head>`, and `<body>` tags to be present, so it is common to reuse a layout file across pages. Your Nuxt `layout.vue` and templates will not include these.
+
+Note the standard HTML templating, and direct access to `<head>`:  
+
+```astro title="src/layouts/Layout.astro"
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Astro</title>
+  </head>
+  <body>
+    <!-- Wrap the slot element with your existing layout templating -->
+    <slot />
+  </body>
+</html>
+```
+
+You may also wish to reuse code from [your Nuxt's page's `head` property](https://nuxtjs.org/docs/configuration-glossary/configuration-head/#the-head-property) to include additional site meta data. Notice that Astro uses neither `vue-meta` nor a component's `head` property creates `<head>` directly. You may import and use components, even within `<head>`, to separate and organize your page content.
+
+### Migrating Pages and Posts
+
+In NuxtJS, your [pages](/en/core-concepts/pages/) live in `/pages`. In Astro, **all your page content must live within `src/`**.
```

Comment:
This could confuse people into thinking that pages can be stored anywhere within `src`, when they can only be stored in `src/pages/` or `src/content`. (I made a similar comment on the Next / Gatsby guides but I'm not sure if we changed them)